crates/generator/src/backend_manage.rs

```diff
@@ -16,6 +24,19 @@ pub struct BackendManage {
     backends: HashMap<H256, Backend>,
 }
 
+/// Get backend name from the validator path
+fn get_backend_name(validator_path: &str) -> Option<BackendName> {
+    if validator_path.contains("meta") {
```

Comment:
We shouldn't use path to detect the script. What happend if the linux user named `meta`?